src/components/ProductListPage/ProductListPage.scss

```diff
@@ -0,0 +1,84 @@
+@import '../../scss/global_styles';
+
+.productListPage {
+  &__line {
+    display: flex;
+    justify-content: space-between;
+    border-top: 2px solid $mercury;
+    margin-bottom: 30px;
+    padding-top: 5px;
+
+    & > * {
```

Comment:
what element(elements) does this selects?

Reply:
all nested elements

Reply:
Is it absolutely necessary to use such selector?